pkg/network/ebpf/c/protocols/http/http.h

```diff
@@ -147,7 +152,7 @@ static __always_inline int http_process(http_transaction_t *http_stack, skb_info
         bpf_map_delete_elem(&http_in_flight, &http_stack->tup);
     }
 
-    return 0;
+    return (packet_type == HTTP_REQUEST || packet_type == HTTP_RESPONSE);
```

Comment:
how is that change related to the bigger scope?
if there's an improvement around http parsing, it should be in a separate PR as this PR is massive and I wish to split massive changes into smaller units